modules/core/quick/src/QuickFormListBuilder.php

```diff
@@ -120,13 +121,13 @@ public function buildRow(EntityInterface $entity) {
       '#markup' => $quick_form->getPlugin()->getLabel(),
     ];
     $row['label'] = [
-      '#markup' => $quick_form->getLabel(),
+      '#markup' => Html::escape($quick_form->getLabel()),
     ];
```

Comment:
TIL I learned more about this!

But looks like we might not need `Html::escape` here and instead just use `#plain_text`. Reading https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Utility%21Html.php/function/Html%3A%3Aescape/8.2.x:

> When constructing [render arrays](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/group/theme_render/10) passing the output of [Html::escape](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Utility%21Html.php/function/Html%3A%3Aescape/10)() to '#markup' is not recommended. Use the '#plain_text' key instead and the renderer will autoescape the text.

Reply:
I think this QuickFormListBuilder is the only place that needs `#plain_text`, all the other places changed in this commit are not explicitly render arrays so `Html::escape` LGTM

Reply:
> But looks like we might not need `Html::escape` here and instead just use `#plain_text`. 

TIL about `#plain_text`! :smile:

Reply:
I pushed this change